pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -830,9 +830,8 @@ public synchronized String toString(LocalDateTime localDateTime) {
       return "-infinity";
     }
 
-    // LocalDateTime is always passed with time zone so backend can decide between timestamp and timestamptz
-    ZonedDateTime zonedDateTime = localDateTime.atZone(getDefaultTz().toZoneId());
-    return toString(zonedDateTime.toOffsetDateTime());
+    OffsetDateTime offsetDateTime = localDateTime.atOffset(ZoneOffset.UTC);
+    return toString(offsetDateTime);
```

Comment:
This renders something like `2019-07-30 09:17:25.676+00`
Why can't this be `2019-07-30 09:17:25.676`, i.e. without zone offset?

Consider use-cases
- inserting `?` bound to `LocalDateTime` to a `timestamp` column. Then zone-less literal would be OK.
- inserting `?` bound to `LocalDateTime` to a `timestamp with time zone` column (which represents point in time in PostgreSQL). Then zone-less literal should probably be coerced on server side to session zone, so zone-less literal should be OK. UTC-zoned literal will likely be not OK as it will represent a different point in time.
- (....)

If this needs to stay this way, please add some explanation.